Presentation.Web/app/components/it-system/it-interface/it-interface-edit.view.html

```diff
@@ -10,9 +10,6 @@ <h3>Snitflade</h3>
                 <li data-ui-sref-active="active">
                     <a data-ui-sref="it-system.interface-edit.main"><img src="/Content/img/Systemforside.svg"></img> Snitfladeforside</a>
                 </li>
-                <li data-ui-sref-active="active">
```

Comment:
Skal hele filen ikke slettes?

Reply:
@mbkstrongminds Har du forholdt dig til denne og de andre kommentarer, som du endnu ikke har svaret på?

Reply:
Det er template filen for side navigations menuen til snitflader. Så det var kun linket til detalje siden der skulle væk